src/main/java/edu/iastate/ece/sd/sdmay2126/runner/selenium/SeleniumOutputCollector.java

```diff
@@ -0,0 +1,16 @@
+package edu.iastate.ece.sd.sdmay2126.runner.selenium;
+
+import edu.iastate.ece.sd.sdmay2126.application.ApplicationOutput;
+import edu.iastate.ece.sd.sdmay2126.application.InvalidApplicationException;
+import edu.iastate.ece.sd.sdmay2126.orchestration.Job;
+
+import javax.annotation.Nonnull;
+
+public interface SeleniumOutputCollector {
```

Comment:
Will this extend a more generic OutputCollector in the future for different types of outputs? Same with the input programmer or are we using those _specifically_ inside of the `SeleniumImplementations` then the others will figure out their own stuff.

Reply:
I figured the `OutputCollector`/`InputProgrammer` would be specific to the Selenium runner. If, eventually, we have some other runner (maybe an SDK runner or a cached runner), they might not have need for the distinction. The main utility in breaking them apart for this one IMO is to separate our work, but functionally it's unnecessary.

Reply:
Makes sense to me.